### ISSUE TITLE
Could we make classesStyles be applied after tagsStyles?

### ISSUE THREAD

**Issue by TheaWork** [archriss/react-native-render-html#35]
Just wanted to say thanks for the great library, we're really impressed with the performance of it vs using a WebView!

We've noticed that, at the moment, tagsStyles override classesStyles - e.g. with the following code, the last-paragraph will come out grey rather than teal, because the styling for 'p' tag is applied after the "last-paragraph" class styling:
```javascript
// props
    tagsStyles: { p: { color: 'grey' } },
    classesStyles: { 'last-paragraph': { color: 'teal' } }

const html = `
    <p class="last-paragraph">Finally, this paragraph is style through the classesStyles prop</p>`;
```

We think it would make more sense to swap this round to be more similar to [CSS Specificity](https://developer.mozilla.org/en-US/docs/Web/CSS/Specificity), where class selectors take precedence over type selectors. What do you think?

I think it might be as simple as just swapping over [lines 324](https://github.com/archriss/react-native-render-html/blob/master/src/HTML.js#L324) and 325 in HTML.js

Let me know if you approve of this change and I can submit a PR for it.  I just wanted to check whether you agree with the idea of the change first!

**Comment by Exilz** [comment]
Hi,
thanks for the feedback !

You're right. Actually, I realized this and recently made the change recently. I'm about to push it on the development branch.

It will land on the next version with some new features and bugfixes.

**Comment by TheaWork** [comment]
Awesome, thank you!

**Comment by Exilz** [comment]
This landed in `3.5.0-rc.1`. Since this release is quite important, testing would be appreciated !
